i18n/zh/docusaurus-plugin-content-docs/current/user-guide/deployments/authentication/ldap.md

```diff
@@ -0,0 +1,79 @@
+# LDAP user provider
```

Comment:
```suggestion
# LDAP User Provider
```
Do we need to translate this title?

Reply:
I'd rather not. It's uncommon to translate "provider".